keyboards/otaku_split/rev0/keymaps/default/keymap.c

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2019 takashiski
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  QMKBEST = SAFE_RANGE,
+  QMKURL
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_wrapper(\
```

Comment:
While this is cool to see here ....

Could you remove this, and use a default layout?  

This is more more of an advanced thing, and we try to actively avoid "clever" things in the default keymaps, since it can cause a lot of confusion for newer/less experienced users.

If you want to use this in a personal keymap, or a "sample" keymap {that isn't "default"), then that is absolutely fine.

Reply:
OK, I will separate default, sample and personal keymap.